home-manager/config/yt-dlp.nix

```diff
@@ -1,8 +1,10 @@
-{ pkgs, config, ... }:
+{ pkgs, config, inputs, ... }:
 
 let
 
-  pkgsUnstable = import <nixpkgs-unstable> {};
+  pkgsUnstable = import inputs.nixpkgs-unstable {
+    system = pkgs.system;
```

Comment:
When using pure evaluation, you gotta pass `system` to nixpkgs when evaluating it. You can easily get the system of the main nixpkgs instance (`pkgs`) using `pkgs.system`